app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -80,6 +80,13 @@ export default {
       },
     };
   },
+  activated() {
```

Comment:
@pranavrajs @scmmishra Is this a proper way? Have less confidence here.

Reply:
I would prefer updated event instead of activated. or a watch event.

Reply:
In case, you want to run this every time the page is visited, you can add it to the navigation guards. See [docs](https://v3.router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards) 

```
beforeRouteEnter (to, from, next) {
  next(vm => {
    // access to component instance via `vm`
  })
}
```

Reply:
done